### ISSUE TITLE
[REQUEST] Chrome for Testing - a new Chrome flavor

### ISSUE THREAD

**Issue by pgagnidze** [Sparticuz/chromium#129]
## What would you like to have implemented?

Can we make use of the new Chrome flavor - Chrome for Testing? Chrome for Testing has been created purely for browser automation and testing purposes. Here is the blog post about it https://developer.chrome.com/blog/chrome-for-testing/

## Why would it be useful?

While Chrome is based on Chromium, they are still different software. It would be nice to have an option for the Chrome.

**Comment by Sparticuz** [comment]
I've been looking into it. The problem is it's huge, like 200+ megs. That being said, I think I do want to somehow integrate @puppeteer/browsers and maybe give people the option. Puppeteer's team came to the conclusion that they didn't want to manage the browser, that's why chrome for testing was made. I just don't know how feasible it is inside a lambda.

**Comment by pgagnidze** [comment]
Good point. I didn't know it was that big package. I have no issue updating my lambda function to use the container image rather than the predefined runtimes, this way, the image size can be up to 10GB (instead of 250MB).
